constructor/nsis/main.nsi.tmpl

```diff
@@ -161,10 +161,13 @@ Page Custom mui_AnaCustomOptions_Show
 !define MUI_FINISHPAGE_TEXT __CONCLUSION_TEXT__
 #endif
 
+#if custom_conclusion
 # Custom conclusion file(s)
 @CUSTOM_CONCLUSION_FILE@
-
+#else
```

Comment:
This is pretty fundamental change. To restore old behavior, our custom conclusion files will have to add `!insertmacro MUI_PAGE_FINISH`, correct?

Reply:
Correct, that was always the intention IIRC, but this sneaked in. Otherwise you can't replace the last page (like the example tries to do), and you get two "Finish" pages which feels weird.

Reply:
Two finish pages may be desirable from a marketing perspective, but I do agree that we shouldn't force a default page. I'm all for more flexibility